src/nidmm/metadata/attributes.py

```diff
@@ -628,8 +634,13 @@
         'resettable': 'No',
         'type': 'ViInt32',
 'documentation': {
-'description': 'For the NI 4060 only, specifies a delay interval after a sample trigger.',
-'table_body': [['0', 'IVI compliant', 'The Sample Interval property is only used when the Sample Trigger is set to Interval.'], ['1', 'Not IVI compliant', 'The Sample Interval property is used as a delay after any type of Sample Trigger.']],
+'description': '''
+For the NI 4060 only, specifies a delay interval after an sample external trigger.
+0
+NIDMM_ATTR_SAMPLE_INTERVAL is only used when the Sample Trigger attribute is set to  INTERVAL.
```

Comment:
Here's an idea that we can open an issue for, if you think it has merit:

In metadata we now have (or will have) the full original C name for the symbol. We could go through all the documentation in all the metadata looking for 'NIDMM_ATTR', 'NIDMM_VAL', 'niDmm_' and when/if found, do the appropriate replacement/linking.

Reply:
Yes, that is my plan for the future. I didn't make it an issue though.

Reply:
Opened #589